workloads/jenkins/scripts/test-dle-deployment.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+source workloads/jenkins/scripts/jenkins-common.sh
+
+set -ex
+
+# Ensure working directory is root
+cd "${ROOT_DIR}" || exit 1
+
+# Deploy a Deep Learning Example
+JENKINS_DLE="${JENKINS_DLE:-pytorch-detection-ssd}"
+JENKINS_DLE_NODEPORT="30888"
+
+if ! which helm; then
+	echo "helm command not available"
+	exit 1
+fi
+
+helm install \
+	--wait \
+	--timeout 1200s \
+	"${JENKINS_DLE}" \
+	workloads/examples/k8s/deep-learning-examples \
+	--set exampleName="${JENKINS_DLE}"
+
+if [ $? -ne 0 ]; then
+	echo "Failed to deploy DLE"
+	exit 1
+fi
+
+sleep 60
```

Comment:
It'd probably be better to copy/paste the polling we do in the other tests, but 60 seconds is probably fine if we are only running this once.